app/models/invoice_subscription.rb

```diff
@@ -13,6 +13,18 @@ class InvoiceSubscription < ApplicationRecord
   monetize :subscription_amount_cents, disable_validation: true, allow_nil: true
   monetize :total_amount_cents, disable_validation: true, allow_nil: true
 
+  scope :order_by_charges_to_datetime, lambda {
+    condition = ActiveRecord::Base.sanitize_sql_for_conditions(
+      <<-SQL
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)